test/unit/RelayServer.test.ts

```diff
@@ -390,68 +472,134 @@ describe('RelayServer tests', function () {
 
         expect(maxPossibleGas.toString()).to.be.eq(
           (
-            fakeEstimationBeforeFees +
-            fakeEstimationBeforeFees * FAKE_GAS_FEE_PERCENTAGE
+            FAKE_ESTIMATION_BEFORE_FEES +
+            FAKE_ESTIMATION_BEFORE_FEES * FAKE_GAS_FEE_PERCENTAGE
           ).toString()
         );
       });
 
-      it('Should charge fees based on transfer value, when transferFeePercentage > 0 and a transfer() is being relayed', async function () {
-        const fakeServerConfigParams = {
-          app: {
-            gasFeePercentage: FAKE_GAS_FEE_PERCENTAGE,
-            transferFeePercentage: FAKE_TRANSFER_FEE_PERCENTAGE,
-          },
-        };
-        sinon.stub(relayServer, 'config').value(fakeServerConfigParams);
+      describe('When transferFeePercentage > 0', function () {
+        it('Should charge fees based on transfer value, when  and a transfer() is being relayed', async function () {
+          const fakeServerConfigParams = {
+            app: {
+              gasFeePercentage: FAKE_GAS_FEE_PERCENTAGE,
+              transferFeePercentage: FAKE_TRANSFER_FEE_PERCENTAGE,
+            },
+          };
+          sinon.stub(relayServer, 'config').value(fakeServerConfigParams);
```

Comment:
Same as above, we could include this code in a `beforeEach` section to avoid code repetition

Reply:
Done.